go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/container-object-storage-interface/spec
+
+go 1.14
```

Comment:
Can this be updated to 1.15.  1.15 is required for k8s 1.19.

Reply:
Fixed